recipe/meta.yaml

```diff
@@ -31,6 +31,15 @@ requirements:
     - xarray >=0.16.1
     - zarr >=2.6
 
+    # Extra Formats (included by default)
+    # [base-imageio]
+    - imageio
+    - imageio-ffmpeg
+    - psutil
```

Comment:
psutil is required here? suprised.

Reply:
just copied what the imageio[ffmpeg] extra states here: https://github.com/imageio/imageio/blob/a5ce49f1604b15f9566e89e476b3198ef57c964f/setup.py#L180

Reply:
Huh. TIL.